backend/build-docker-and-push

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eu
+
+GIT_SHORT_REF=$(git rev-parse --short HEAD)
+DOCKER_TAG="strapi:$GIT_SHORT_REF"
+# iFixit AWS ECR for strapi
+ECR_TAG="884681002735.dkr.ecr.us-east-1.amazonaws.com/$DOCKER_TAG"
+
+echo "Building docker image"
+docker build -t "$DOCKER_TAG" .
```

Comment:
I think we should probably use Kaniko here? To avoid docker-in-docker?

Reply:
Do we currently plan to run this from Docker? If not, I hear you, but I think we should wait to solve that until it's an issue.

Reply:
Oh, im not sure what context this runs in is all. Just want to make sure we avoid coupling to docker _if_ we're going to run this inside docker.